ArtOfIllusion/src/artofillusion/PluginRegistry.java

```diff
@@ -65,50 +68,26 @@ public static void scanPlugins()
       }
       catch (Exception ex)
       {
-        System.err.println("*** Exception loading plugin file "+file);
+        results.add("Error loading plugin file: " + file);
+        System.err.println("*** "+file);
         ex.printStackTrace(System.err);
       }
     }
-    processPlugins(jars);
+    processPlugins(jars,results);
+    return results;
   }
 
-  /**
-   * Process a set of ClassLoaders corresponding to jar files, read in their indices,
-   * and record all plugins contained in them.
-   */
 
-  public static void scanPlugins(List<ClassLoader> loaders)
-  {
-    HashSet<JarInfo> jars = new HashSet<JarInfo>();
-    for (ClassLoader loader : loaders)
-    {
-      try
-      {
-        jars.add(new JarInfo(loader));
-      }
-      catch (IOException ex)
-      {
-        // Not a zip file.
-      }
-      catch (Exception ex)
-      {
-        System.err.println("*** Exception loading plugin classloader");
-        ex.printStackTrace(System.err);
-      }
-    }
-    processPlugins(jars);
-  }
```

Comment:
Another public method. Where did this functionality get moved to?

Reply:
Just removed. Did not find any usage for it